### ISSUE TITLE
Translation not working properly any longer in admin console

### ISSUE THREAD

**Issue by r3st** [keycloak/keycloak#17503]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

translations

### Describe the bug

Activating localization for a realm do not work properly. After login with different locale, then only some parts of user interface will be translated to given language. 

Example: side menu translation after choosing "Deutsch" shows for user "Benutzer", but for groups "groups"

http://localhost:8080/resources/hxnqy/admin/keycloak.v2/resources/en/groups.json is working normally, but for example http://localhost:8080/resources/hxnqy/admin/keycloak.v2/resources/de/groups.json the second request returns only:
```
{
    "createGroup": "Gruppe erstellen",
    "deleteGroup": "Gruppe löschen",
    "members": "Mitglieder",
    "create": "Erstellen",
    "email": "Email",
    "lastName": "Nachname",
    "firstName": "Vorname",
    "attributes": "Attribute"
}
```

Translations which are missing, located in earlier versions in admin-messages_de.properties, which now is missing in 21.0.1.
`/theme/base/admin/messages/admin-messages_de.properties:groups=Gruppen`

Also creating a messages_de.properties in custom theme path not working as expected, no change on request given URLs or in user interface.

### Version

15.1.1,20.0.3,21.0.1

### Expected behavior

After changing locale to "Deutsch" in login mask and login to "Keycloak Administration UI" menu is fully translated as in earlier versions, for example 15.1.1

### Actual behavior

After changing locale to "Deutsch" in login mask and login to "Keycloak Administration UI" menue is not fully translated. For example, the groups section.

### How to Reproduce?

1. activate Internationalization for master realm
2. add "Deutsch" to Supported locales
3. logout master realm
4. on login page, choose "Deutsch" and login
5. Admin UI not fully translated as in earlier Versions

**Comment by edewit** [comment]
There are a lot more and new keys in the new console, we have tried to migrate as much of the old translations files, but not everything.

**Comment by r3st** [comment]
@edewit Thanks for your Answer. Is there at this time a possibility to add custom translation? And where to find global key list?

**Comment by edewit** [comment]
yes atm the files are plain json in this folder https://github.com/keycloak/keycloak/tree/main/js/apps/admin-ui/public/resources there is also the possibility to add keys with under the internationalisation tab of realm settings
![image](https://user-images.githubusercontent.com/51133/226608277-f04719e7-3a16-4b93-97dd-e5ca48dc2196.png)

**Comment by r3st** [comment]
Thanks for help! This will help me to build a solution at this point

**Comment by ssilvert** [comment]
@r3st Can we close this issue?
